marklogic-data-hub/src/main/resources/ml-modules/root/com.marklogic.hub/config.sjs

```diff
@@ -8,5 +8,7 @@ module.exports = {
   HUBLOGLEVEL: "%%mlHubLogLevel%%",
   FLOWOPERATORROLE: "%%mlFlowOperatorRole%%",
   FLOWDEVELOPERROLE: "%%mlFlowDeveloperRole%%",
+  DATAHUBOPERATORROLE: "%%mlDataHubOperatorRole%%",
```

Comment:
I think it's better not to define these here, as it implies that they could vary per environment. But we don't want them to vary. I think constants defined elsewhere is fine to reduce duplication of the string value, but I wouldn't consider that necessary either. These role names are intended to be written in stone and never to change.

Reply:
moved to the constants library.